src/PaymentExporter.php

```diff
@@ -28,6 +28,17 @@ public function toJson(\Payment $payment) {
     $data['method_specific'] = $payment->method->title_specific;
     $data['method_generic'] = $payment->method->title_generic;
     $data['controller'] = $payment->method->controller->name;
+    $data['line_items'] = [];
+    foreach ($payment->line_items as $name => $line_item) {
+      $data['line_items'][$name] = [
+        'unit_amount_subunits' => (int) round($line_item->amount * $subunits),
```

Comment:
Why no unrounded `$line_item->amount`?

2 line items with an amount of e.g. 1.333 would yield different results in the data:
2 x 1.333 rounds to 267 in the `total_amount` above and the `total_amount_subunits`, but only adds up to 266 when calculating unit_amount_subunits * quantity.

Reply:
’m extending campaignion_logcrm for what logcrm expects and the logCRM side is already merged. So it would need changing as well when we want to support additional use-cases.

1. I didn’t think there was an actual use case where line items would have amounts that can’t be represented as integers in the smallest currency unit.
2. I’d prefer to avoid the complexity of dealing with floats in money.